src/backends/pgfplots.jl

```diff
@@ -175,52 +175,55 @@ end
 
 function _pgfplots_get_plot_kwargs(plt)
     kwargs = KW()
-    kwargs[:style] = ""
+    kwargs[:style] = []
     _pgfplots_get_linestyle!(kwargs, plt)
     _pgfplots_get_marker!(kwargs, plt)
     _pgfplots_get_series_color!(kwargs, plt)
     _pgfplots_get_label!(kwargs, plt)
+    kwargs[:style] = join(kwargs[:style], ',')
     kwargs
 end
 
+
+
 function _pgfplots_axis(plt_series)
     line_type = plt_series[:linetype]
     plt_kwargs = _pgfplots_get_plot_kwargs(plt_series)
```

Comment:
this line returns a `plt_kwargs[:style]` which has been joined... (continued below)